config/common/csi/daemonset.yaml

```
@@ -0,0 +1,172 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    dynatrace.com/operator: dynatrace
+  name: dynatrace-oneagent-csi-driver
+  namespace: dynatrace
+spec:
+  revisionHistoryLimit: 10
+  selector:
+    matchLabels:
+      internal.oneagent.dynatrace.com/app: csi-driver
+      internal.oneagent.dynatrace.com/component: csi-driver
+  template:
+    metadata:
+      annotations:
+        kubectl.kubernetes.io/default-logs-container: driver
```

This annotation is [deprecated](https://kubernetes.io/docs/reference/labels-annotations-taints/#kubectl-kubernetes-io-default-container) in favor of the following one:
```suggestion
        kubectl.kubernetes.io/default-container: driver
```